lib/prisma.ts

```
@@ -0,0 +1,19 @@
+
+const prismaClientSingleton = () => {
+  return new PrismaClient()
+}
+
+type PrismaClientSingleton = ReturnType<typeof prismaClientSingleton>
+
+declare global {
+  var prisma: PrismaClientSingleton | undefined
+}
+
+const prisma = globalThis.prisma ?? prismaClientSingleton()
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)